db/institutions.yml

```diff
@@ -325,6 +325,17 @@ mpi:
       name: Institute for Solid State Physics
   name: Max Planck Institute
   state: ''
+msk:
+  aka:
+    - MSK
+    - Sloan Kettering
+    - Memorial Sloan Kettering Cancer Center
+  city: New York
+  country: USA
+  name: Memorial Sloan Kettering Cancer Center
```

Comment:
we need a department.  It is a conditional requirement though not as such in the schema. Maybe we should make it required

Reply:
could be a center or a computation group or something